app/models/cloud9_invite.rb

```diff
@@ -0,0 +1,16 @@
+class Cloud9Invite < ApplicationRecord
+  validates :email, presence: true
+  validates :email, uniqueness: true
+  validates :email, email: true
+
+  before_create :invite_member
+
+  def invite_member
+    Cloud9Client::Team.invite_member(
+      Rails.application.secrets.cloud9_team_name,
+      email
+    )
+  rescue
+    errors.add(:invite_member_error, 'Couldn\'t invite member')
+  end
```

Comment:
- I'd make this method private and name it something like `send_invite`, so it's clearer that some sort of external action is happening.
- `'Couldn\'t invite member'` can also be changed to `"Couldn't invite member"'
- Using `rescue` without specifying which exception you want to catch is very dangerous because it catches _everything_. What if something unexpected happens?